ci/check-documentation.py

```diff
@@ -114,8 +114,25 @@ def complain(message):
             if md_path in index_child_md_links:
                 referenced_md_files.add(target_path)
 
-        # check for existence of links to README.md files of models and demos
+        # check <omz_dir> link validity
+
+        for link in sorted([link for link in doc_page.omz_references() if link.startswith('<omz_dir>')]):
+            file_path = Path(link.replace('<omz_dir>', str(OMZ_ROOT)))
+
+            try:
+                file_relative_path = file_path.relative_to(OMZ_ROOT)
+            except ValueError:
```

Comment:
How can this exception ever occur? `file_path` always begins with `OMZ_ROOT` by construction, so `relative_to` should never fail.

Moreover, it seems counterproductive to add `OMZ_ROOT` to the path and then to try to remove it again. It would be more straightforward to initialize `file_relative_path` first (by stripping the `<omz_dir>/` prefix from the reference) and _then_ build `file_path`.